snewpdag/plugins/EvalMap.py

```diff
@@ -46,17 +47,17 @@ def compare(self, keys, tdelay):
     tdelay = time offsets in s, shape (nkeys,)
     Return chi2-like measure.
     """
-    # Choose TimeHist with coarsest binning to set t=0
+    # Choose Hist1D with coarsest binning to set t=0
     # so we don't have to rebin it.
-    # if there were no TimeHists, then we'll just use first.
+    # if there were no Hist1D, then we'll just use first.
     kc = None
     max_width = 0.0
     for k in keys:
       if kc == None:
         kc = k
       v = self.cache[k]
-      if isinstance(v, TimeHist):
-        bin_width = v.duration / v.nbins # seconds
+      if isinstance(v, Hist1D):
+        bin_width = v.xwidth / v.nbins # seconds
```

Comment:
Previously, I did not see xwidth and nbins when using Hist1D for Uniform generator, but indeed I see they are here, so maybe you can skip that comment